internal/limitrange/limitrange.go

```diff
@@ -17,3 +21,29 @@ func NewConfig(lri corev1.LimitRangeItem, resource corev1.ResourceName) Config {
 
 	return l
 }
+
+//LimitRange provides an implementation of the LimitRanger interface defined in admission. This implementation is designed to provider a generic config sourcing tool for all mutation handlers.
+type LimitRange struct {
+	lister corev1Listers.LimitRangeLister
+}
+
+//LimitRangeConfig takes a namespace string and returns a Config for memory or a nil if no limit range of type Container is found in the namespace. It returns a non-nil error if there is an error sourcing data from the cluster api or the namespace name is empty
+func (lr *LimitRange) LimitRangeConfig(namespace string) (*Config, error) {
+	if namespace == "" {
+		return nil, fmt.Errorf("Invalid namespace: %q", namespace)
```

Comment:
nit: should be lower case
```suggestion
		return nil, fmt.Errorf(invalid namespace: %q", namespace)
```